valexa/gui/profile_widget.py

```diff
@@ -34,19 +34,49 @@ def __init__(self, profile: Profile):
-        table.setRowCount(len(self.profile.levels))
-        table.setColumnCount(6)
-        for (index, l) in enumerate(self.profile.levels):
-            table.setItem(index, 0, QTableWidgetItem(f'{l.introduced_concentration:.3f}'))
-            table.setItem(index, 1, QTableWidgetItem(f'{l.calculated_concentration:.3f}'))
-            table.setItem(index, 2, QTableWidgetItem(f'{l.bias:.3f}'))
-            table.setItem(index, 3, QTableWidgetItem(f'{l.relative_bias:.2f}'))
-            table.setItem(index, 4, QTableWidgetItem(f'{l.recovery:.1f}'))
-            table.setItem(index, 5, QTableWidgetItem(f'[{l.abs_tolerance[0]:.2f},{l.abs_tolerance[1]:.2f}]'))
-
-        table.setHorizontalHeaderLabels(
-            ["Concentration", "Calculated concentration", "Absolute bias", "Relative biais (%)", "Recovery (%)", "Tolerance Interval"])
+        table.setRowCount(len(self.profile.levels)+2)
+        table.setColumnCount(8)
+
+        table.setSpan(0, 0, 2, 1)
+        table.setSpan(0, 1, 2, 1)
+        table.setSpan(0, 2, 1, 2)
+        table.setSpan(0, 4, 1, 2)
+        table.setSpan(0, 6, 1, 2)
+
+        table.setItem(0, 0, QTableWidgetItem('Concentration'))
+        table.setItem(0, 1, QTableWidgetItem('Mesuré'))
+        table.setItem(0, 2, QTableWidgetItem('Exactitude'))
+        table.setItem(0, 4, QTableWidgetItem('Précision'))
+        table.setItem(0, 6, QTableWidgetItem('Justesse'))
+
+        table.setItem(1, 2, QTableWidgetItem('Biais absolue'))
+        table.setItem(1, 3, QTableWidgetItem('Biais relatif'))
+
+        table.setItem(1, 4, QTableWidgetItem('Répétablité (%RSD)'))
+        table.setItem(1, 5, QTableWidgetItem('Précision intermédaire (%RSD)'))
+
+        table.setItem(1, 6, QTableWidgetItem('Intervales de tolérance'))
+        table.setItem(1, 7, QTableWidgetItem('Limite de tolérance relative'))
+
+
+        for (index, list_item) in enumerate(self.profile.levels):
+            table.setItem(index + 2, 0, QTableWidgetItem(f'{list_item.introduced_concentration:.3f}'))
+            table.setItem(index + 2, 1, QTableWidgetItem(f'{list_item.calculated_concentration:.3f}'))
+            table.setItem(index + 2, 2, QTableWidgetItem(f'{list_item.bias:.3f}'))
+            table.setItem(index + 2, 3, QTableWidgetItem(f'{list_item.relative_bias:.2f}'))
+            table.setItem(index + 2, 4, QTableWidgetItem(f'{list_item.repeatability_std_pc:.3f}'))
+            table.setItem(index + 2, 5, QTableWidgetItem(f'{list_item.inter_series_std_pc:.3f}'))
+            #table.setItem(index+1, 4, QTableWidgetItem(f'{list_item.recovery:.1f}'))
+            table.setItem(index + 2, 6,
+                          QTableWidgetItem(f'[{list_item.abs_tolerance[0]:.2f},{list_item.abs_tolerance[1]:.2f}]'))
+            table.setItem(index + 2, 7,
+                          QTableWidgetItem(f'[{-100 + list_item.rel_tolerance[0]:.2f},{-100 + list_item.rel_tolerance[1]:.2f}]'))
+
+        #table.setHorizontalHeaderLabels(
```

Comment:
If this block of code is no more used it should not remain in the code as a smelly comment.